### ISSUE TITLE
SSH not found for argo-cd container

### ISSUE THREAD

**Issue by SoulKyu** [bitnami/containers#1652]
### Name and Version

bitnami/argo-cd:2.4.8-debian-11-r0

### What steps will reproduce the bug?

1. Install the whole Argocd application
2. Add a new repository using ssh repo
3. Try to create an application using this repo

### What is the expected behavior?

The repo should be downloaded and the application created

### What do you see instead?

The repo isn't downloaded due to SSH not found:

```
time="2022-08-03T08:37:10Z" level=info msg=Trace args="[git fetch origin --tags --force]" dir=/tmp/_argocd-repo/c959c106-5144-4b7f-a8c4-a07d05b07132 operation_name="exec git" time_ms=3.250127
time="2022-08-03T08:37:10Z" level=error msg="finished unary call with code Internal" error="rpc error: code = Internal desc = Failed to fetch default: `git fetch origin --tags --force` failed exit status 128: ssh -i /dev/shm/2956503429 -o StrictHostKeyChecking=yes -o UserKnownHostsFile=/app/config/ssh/ssh_known_hosts: 1: ssh: not found\nfatal: Could not read from remote repository.\n\nPlease make sure you have the correct access rights\nand the repository exists." grpc.code=Internal grpc.method=GenerateManifest grpc.service=repository.RepoServerService grpc.start_time="2022-08-03T08:37:09Z" grpc.time_ms=1015.716 span.kind=server system=grpc
```

![image](https://user-images.githubusercontent.com/30684712/182565111-cff45304-cc5a-4c7a-9745-8b83d8e0aa8a.png)

**Comment by SoulKyu** [comment]
https://github.com/bitnami/containers/pull/1654

**Comment by SoulKyu** [comment]
UP, this should be an easy fix ...

**Comment by aoterolorenzo** [comment]
Hi @SoulKyu, thank you for taking the time to submit the issue and the Pull request. The issue is now solved :)

**Comment by beauvankirk** [comment]
Is there something that needs to happen to trigger publishing a new image tag with this fix?

**Comment by aoterolorenzo** [comment]
@beauvankirk Should be already out. I just force it so should be finally out in a few hours.

**Comment by beauvankirk** [comment]
I could be missing something but it looks like something is still preventing successful publish. Digging through the GitHub Actions log I did find two failed runs for this image, the last of which seems to maybe line up time-wise with your comment:
https://github.com/bitnami/containers/actions/runs/2866971110
Is the message "Project already has the associated issue" stemming from the fact that I commented on an already closed issue? (i.e. there was already a successful Action execution for this issue).
I'm not familiar with the setup though, and I'm not sure what the "Organizing cards based on comments" means.

**Comment by aoterolorenzo** [comment]
@beauvankirk I'm afraid that the pipeline for argo-cd is on-hold since the changes are to be included in new version/s (from [v2.4.9](https://github.com/argoproj/argo-cd/releases/tag/v2.4.9)) that need a manual action to get through.

**Comment by beauvankirk** [comment]
Thanks for the info. I think openssh-client would definitely be sufficient and probably be better since 'ssh' is just a metapackage containing openssh-client and openssh-server. The upstream image does not appear to have openssh-server either:

![image](https://user-images.githubusercontent.com/20178486/185501638-d91c724d-4377-4819-b43f-b7078c9ad14b.png)

**Comment by aoterolorenzo** [comment]
Yeah, totally agree. In fact there is already a task done adding `openssh-client` instead of `ssh`. Will be out once the next release be out. Thank you!

**Comment by carrodher** [comment]
The `ssh` binary is included in the latest published image once the new version (`2.4.11`) was released, see changes at https://github.com/bitnami/containers/pull/4810:
```console
$ docker run -it bitnami/argo-cd:latest ssh
usage: ssh [-46AaCfGgKkMNnqsTtVvXxYy] [-B bind_interface]
           [-b bind_address] [-c cipher_spec] [-D [bind_address:]port]
           [-E log_file] [-e escape_char] [-F configfile] [-I pkcs11]
           [-i identity_file] [-J [user@]host[:port]] [-L address]
           [-l login_name] [-m mac_spec] [-O ctl_cmd] [-o option] [-p port]
           [-Q query_option] [-R address] [-S ctl_path] [-W host:port]
           [-w local_tun[:remote_tun]] destination [command]
```

This was done by including the `openssh-client` package instead of `ssh`
